index.html

```diff
@@ -570,6 +570,32 @@ <h2 id="constrainable-properties">
                 again.</p>
               </td>
             </tr>
+            <tr id="def-constraint-suppressLocalAudioPlayback">
+              <td><dfn>suppressLocalAudioPlayback</dfn></td>
+              <td>{{ConstrainBoolean}}</td>
+              <td>
+                <p>As a setting, this value indicates whether or not the user
+                agent is applying <a>local audio playback suppression</a> to
+                the source.</p>
+                <p>As a constraint, this value is only meaningful if the user
+                selects capturing a [=browser=] [=display surface=]. In that
+                case, a value of <code>true</code> indicates that the user
+                agent SHOULD perform <a>local audio playback suppression</a>
+                on the captured [=browser=] [=display surface=].</p>
+                <p>When <dfn>local audio playback suppression</dfn> is applied,
+                the user agent SHOULD stop relaying audio to the local speakers,
+                but that audio MUST still captured by any ongoing audio-capturing
+                capture-sessions.</p>
```

Comment:
I'd still like the following sentence in about observability in JavaScript, to clarify what should be implemented, since I see two interpretations otherwise:
 1. Speaker output is muted at the tab-level in the browser, with no behavioral impact to the JS documents in it.
 2. Audio observably stops in APIs like tracks and web audio.  

I want to clarify we mean #1. E.g. any graphical audio visualizer a page makes up, keeps animating, and any audio it sends over a peer connection will still be audible at the peer.

```suggestion
                capture-sessions. This suppression MUST not be observable to the document,
                other than through its setting value.</p>
```

Reply:
Reworded this somewhat.
1. MUST NOT
2. Clarified observability to the captured document (none at all) vs. to the capturing document (can only re-examine if it has applied the constraint; not if it has/had an effect).